domains/tunnel.riviox.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "riviox",
+           "email": "rivioxyt@hotmail.com",
+           "discord": "1200520669570539620"
+        },
+    
+        "record": {
+            "CNAME": "https://tough-mink-modest.ngrok-free.app"
```

Comment:
`CNAME` cannot contain protocol (https://)

Reply:
What 21Z said above. Remove the protocol.